### PR TITLE
xcb-proto: Upgrade version to 1.16

### PR DESCRIPTION
**Pull request by xordux**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
- xcb-proto: Upgrade version to 1.15
- Removed Python 3.9 patch (not required for newer version of xcb-proto) 

###### Change Log  

- xcb-proto

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: [487082](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=487082&view=results)
- Local build.

### CODE REVIEW

File: SPECS/xcb-proto/xcb-proto.spec [hunk 1]
```diff
@@ -2,20 +2,16 @@
 %define debug_package %{nil}
 Summary:        XCB protocol descriptions
 Name:           xcb-proto
-Version:        1.13
-Release:        16%{?dist}
+Version:        1.15
```
Comment (mfrw) [inline SPECS/xcb-proto/xcb-proto.spec:5]: There seems to be a `1.16` release as well

**Comment by ameyapu** [comment]
LGTM

File: SPECS/xcb-proto/xcb-proto.spec (hunk 1 above)
Comment (xordux) [reply SPECS/xcb-proto/xcb-proto.spec:5]: Sure @mfrw  I will try it, but I could only find 1.15 as latest here: https://xcb.freedesktop.org/dist/ 
Do you happen to know any other source of xcb packages where I can get that 1.16 version?

**Review by ameyapu — APPROVED** [review]
Looks good to me.

File: SPECS/xcb-proto/xcb-proto.spec (hunk 1 above)
Comment (mfrw) [reply SPECS/xcb-proto/xcb-proto.spec:5]: Yes, https://xorg.freedesktop.org/archive/individual/proto/ 
has 1.16

File: SPECS/xcb-proto/xcb-proto.spec [hunk 2]
```diff
@@ -2,20 +2,16 @@
...
-Version:        1.13
-Release:        16%{?dist}
+Version:        1.16.0
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://xcb.freedesktop.org/
-Source0:        https://xcb.freedesktop.org/dist/%{name}-%{version}.tar.bz2
+Source0:        https://xorg.freedesktop.org/archive/individual/proto/%{name}-%{version}.tar.gz
```
Comment (arc9693) [inline SPECS/xcb-proto/xcb-proto.spec:11]: URL update is not needed. Additionally, adding .0 seems like a new addition on this URL for the tarball versions. Not sure, if we should deviate?

**Review by arc9693 — APPROVED** [review]

**Pull request merged**